docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md

```diff
@@ -0,0 +1,62 @@
+# 2024, Summer - OFA Admin Experience
+
+**Jump To:**
+
+* [Background](#background)
+* [What we did & who we talked to](#what-we-did--who-we-talked-to)
+* [What we learned](#what-we-learned)
+* [Next steps](#next-steps)
+
+***
+
+## Background
+
+The Django Admin Console (DAC) is an internal tool tailored for our OFA System Admin and DIGIT Team users. It allows for access to the Postgres database & system logs, managing user permissions, and has been increasingly adopted to provide the DIGIT team with quick insights into data file errors and STT submissions.&#x20;
```

Comment:
i think its important to clarify here that the permissions for each group are different. I wouldn't want to misrepresent privileged access. digit team's access to DAC is limited/not privileged. and actually needs to be locked down further (follow-on ticket)